packages/arb-avm-cpp/avm_values/src/vmValueParser.cpp

```diff
@@ -16,102 +16,82 @@
 
 #include <avm_values/vmValueParser.hpp>
 
-#include <bigint_utils.hpp>
+#include <nlohmann/json.hpp>
 
 #include <fstream>
 
-std::vector<char> getContractData(const std::string& contract_filename) {
-    std::ifstream contract_file(contract_filename,
-                                std::ios::in | std::ios::binary);
-    if (!contract_file.is_open()) {
-        return {};
+namespace {
+
+value value_from_json(const nlohmann::json& j,
```

Comment:
im a fan of having variable names be more than just a letter to be more descriptive

Reply:
Totally agreed. Updated all the short variable names in this class